src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/Jira.kt

```diff
@@ -11,10 +12,10 @@ import org.apache.logging.log4j.Logger
 import java.net.URI
 import java.util.concurrent.Executors
 
-class Jira(
+class Jira private constructor(
     private val nodes: List<StartedNode>,
     val jiraHome: RemoteLocation,
-    val database: RemoteLocation?,
+    val database: RemoteLocation,
     val address: URI,
     val jmxClients: List<JmxClient> = emptyList()
```

Comment:
When we are breaking API did you consider unshipping JMX clients from the core Jira formulas?